benchmark.py

```diff
@@ -30,12 +30,14 @@
 parser.add_argument('--onnx')
 parser.add_argument('--stft-mode', choices = ['conv', ''], default = '')
 parser.add_argument('-B', type = int, default = 256)
-parser.add_argument('-T', type = int, default = 5.12)
+parser.add_argument('-T', type = float, default = 5.12)
```

Comment:
Why is non-integer float T needed?

Reply:
For cases when needed to test divisibility.